### PR TITLE
LPS-163845 Send email to invited user

### PR DESCRIPTION
**Pull request by pei-jung**

### CODE REVIEW

**Comment by drewbrokke** [comment]
@patriciajeanperez is this failure related?

@pei-jung do you think there are any functional test updates necessary?

**Comment by pei-jung** [comment]
@drewbrokke yes, we'll need functional tests update for this since we changed the creation menu from a button to a dropdown for the assign account users. We'll also need new functional test for the new invite users feature.

/cc @patriciajeanperez

**Comment by patriciajeanperez** [comment]
I will send a new PR with the fixes and test @pei-jung @drewbrokke

**Comment by patriciajeanperez** [comment]
ci:test:account

**Review by drewbrokke — APPROVED** [review]

File: modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountEntryUserRelLocalServiceImpl.java [hunk 1]
```diff
@@ -444,6 +503,92 @@ private String[] _getAccountDomains(long accountEntryId) {
...
+					"emailAddress", emailAddress
+				).toString(),
+				null, serviceContext);
+
+			Group guestGroup = _groupLocalService.getGroup(
+				inviter.getCompanyId(), GroupConstants.GUEST);
+
+			String url = PortletURLBuilder.create(
+				PortletURLFactoryUtil.create(
+					serviceContext.getRequest(),
```
[the remark this answers is not in the document]
Comment (pei-jung) [reply modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountEntryUserRelLocalServiceImpl.java:536]: That'd be nice, but I haven't found a way to create a portlet URL without a request object. If you have any idea, please let me know.

[the remark this answers is not in the document]
Comment (drewbrokke) [reply modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountEntryUserRelLocalServiceImpl.java:536]: Makes sense. I saw what you had to do in the test case and realized this is "hidden" api, but I can't see a way around it right now. Thankfully in the real-world usage it's probably not an issue. Perhaps at the very least we document this unique aspect, and handle/log/throw in the case when no request is found?

**Comment by patriciajeanperez** [comment]
Running Account tests in a separate PR that includes a temporary fix to unblock the tests due to a [bug](https://issues.liferay.com/browse/LPS-166676): https://github.com/patriciajeanperez/liferay-portal/pull/671

**Comment by patriciajeanperez** [comment]
Resent: https://github.com/liferay-usm/liferay-portal/pull/807

**Pull request closed without merging**